astropy/coordinates/builtin_frames/gcrs.py

```diff
@@ -3,8 +3,8 @@
 
 from astropy import units as u
 from astropy.utils.decorators import format_doc
-from astropy.coordinates.attributes import (TimeAttribute,
-                          CartesianRepresentationAttribute)
+from astropy.coordinates.attributes import (
+    TimeAttribute, CartesianRepresentationAttribute)
```

Comment:
Since we have two lines anyway, I prefer the previous format, assuming to align  `CartesianRepresentationAttribute`, too. Or would that be over the line length limit?